lib/PostEditorBlocks/MarketingOptinBlock.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace MailPoet\PostEditorBlocks;
+
+use Automattic\WooCommerce\Blocks\Integrations\IntegrationInterface;
+use MailPoet\Config\Env;
+
+/**
+ * Class MarketingOptinBlock
+ *
+ * Class for integrating marketing optin block with WooCommerce Checkout.
+ */
+class MarketingOptinBlock implements IntegrationInterface {
+  /** @var array */
+  private $options;
+
+  public function __construct(
+    array $options
+  ) {
+    $this->options = $options;
+  }
+
+  /**
+   * The name of the integration.
+   *
+   * @return string
+   */
+  public function get_name() { // phpcs:ignore
+    return 'mailpoet';
+  }
+
+  /**
+   * Register block scripts and assets.
+   */
+  public function initialize() {
+    $script_asset_path = Env::$assetsUrl . '/dist/js/marketing_optin_block/marketing-optin-block-frontend.asset.php';
+    $script_asset      = file_exists($script_asset_path)
```

Comment:
In our coding standards we only have a single space on each side of `=`, could you please remove the extra spaces?

Reply:
Do you have a phpcs/phpcbf config available so formatting is not a manual process?

Reply:
Yes, we have (./do qa command uses it), but it seems this is not covered by our ruleset even though we thought it was :). Let's leave it as is. We will revisit it later.